supplementary_style_guide/glossary_terms_conventions/general_conventions/t.adoc

```diff
@@ -139,7 +139,7 @@
 [discrete]
 [[throughput]]
 ==== image:images/yes.png[yes] throughput (noun)
-*Description*: "Throughput" is the amount of data transferred from one place to another or processed in a specified amount of time. Data transfer rates for disk drives and networks are measured in terms of throughput. Typically, throughput is measured in kbps, Mbps, or Gbps. See the _IBM Style Guide_ for more information about using measurements and abbreviations.
+*Description*: _Throughput_ is the amount of data transferred from one place to another or processed in a specified amount of time. Data transfer rates for disk drives and networks are measured in terms of throughput. Typically, throughput is measured in kbps, Mbps, or Gbps. See the _IBM Style Guide_ for more information about using measurements and abbreviations.
```

Comment:
Probably out of scope, but I'll mark as optional: should kbps, Mbps, or Gbps be surrounded by backticks?

Reply:
I don't necessarily think so, so I'll leave it as is.